src/views/datasets/hospitalization-hotspot/dashboard.vue

```diff
@@ -0,0 +1,399 @@
+<template>
+  <DashboardCard width="full">
+    <template #subtitle>
+      This tool shows us places where people who have gone to the hospital with
+      COVID-19 live. This tool focuses on places where more people have gone to
+      the hospital compared to state levels. We call these places
+      <em>hotspots</em>. You can use this information to find where the hotspots
+      are in our state and take steps to help eliminate them.
+    </template>
+  </DashboardCard>
+
+  <DashboardCard width="full" :height="1">
+    <template #content>
+      <ControlPanel
+        :drop-downs="dropDowns"
+        :init-value="controls"
+        @selected="updateControls"
+      />
+    </template>
+  </DashboardCard>
+
+  <DashboardCard width="full" :height="5">
+    <template #title>{{
+      activeCluster.name
+        ? `Community: ${activeCluster.name}`
+        : "Select a community to explore"
+    }}</template>
+
+    <template #top-right>
+      <button
+        :disabled="!activeCluster.name"
+        class="zoom-button button is-link"
+        @click="zoomed = !zoomed"
+      >
+        <span class="icon">
+          <i
+            class="fas"
+            :class="[zoomed ? 'fa-search-minus' : 'fa-search-plus']"
+          ></i>
+        </span>
+        <span>{{
+          zoomed ? "Zoom Back Out" : "Zoom to Selected Community"
+        }}</span>
+      </button>
+    </template>
+
+    <template #subtitle>
+      This map shows hotspots where more people have gone to the hospital with
+      COVID-19 compared to state levels. Darker areas show higher rates of
+      hospital stays among <strong>{{ controls.focusStat.name }}</strong
+      >. Areas with dashes mean there is not enough information. Select a
+      community, click the <em>Zoom to Community</em> button, and scroll down to
+      learn more.
+    </template>
+
+    <template #content>
+      <div class="map-container">
+        <ColdMap
+          :geo="data.geo"
+          :stats="data.stats"
+          :filter-town="controls.town"
+          :focus-stat="controls.focusStat"
+          :initial-active-cluster="dashboardActiveCluster"
+          :map-type="'hot'"
+          class="is-absolute"
+          @new-active-cluster-id="updateCluster"
+        />
+      </div>
+      <!-- TODO change to make this keep the space like done with 'what is age-adjusted rate' in other PR -->
```

Comment:
What does this TODO mean?

Reply:
I believe i just forgot to remove the TODO when something else changed!

Reply:
Just an erroneous copy of a TODO that has been done!